components/script_traits/lib.rs

```diff
@@ -559,6 +561,8 @@ pub struct InitialScriptState {
     pub webvr_chan: Option<IpcSender<WebVRMsg>>,
     /// The Webrender document ID associated with this thread.
     pub webrender_document: DocumentId,
+    /// FIXME(victor): The Webrender API sender in this constellation's pipeline
```

Comment:
@ceyusa what needs to be fixed here?

Reply:
The same I commented above, not a fixme but a request to rethink this approach,